guacamole/src/main/webapp/app/form/templates/passwordField.html

```diff
@@ -6,6 +6,7 @@
            ng-trim="false"
            guac-focus="focused"
            autocorrect="off"
-           autocapitalize="off"/>
+           autocapitalize="off"
+           autocomplete="new-password"/>
```

Comment:
I'm not immediately sure how to deal with this, but I think this may cause trouble on login pages which use this field. In those cases, we're looking for an existing password, and "new-password" might not be the correct choice.

In your testing, did this disrupt login behavior at all?